File: c#/src/Docmost.Sdk/Models/Space.cs
namespace Docmost.Sdk.Models;

public class Space
{
    [System.Text.Json.Serialization.JsonPropertyName("id")]
    public string? Id { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("name")]
    public string? Name { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("slug")]
    public string? Slug { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("description")]
    public string? Description { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("logo")]
    public string? Logo { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("defaultRole")]
    public string? DefaultRole { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("visibility")]
    public string? Visibility { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("settings")]
    public string? Settings { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("creatorId")]
    public string? CreatorId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("workspaceId")]
    public string? WorkspaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("createdAt")]
    public string? CreatedAt { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("updatedAt")]
    public string? UpdatedAt { get; set; }

}

File: go/docmost/models/space.go
package models

// Space from OpenAPI.
type Space struct {
	Id *string `json:"id,omitempty"`
	Name *string `json:"name,omitempty"`
	Slug *string `json:"slug,omitempty"`
	Description *string `json:"description,omitempty"`
	Logo *string `json:"logo,omitempty"`
	DefaultRole *string `json:"defaultRole,omitempty"`
	Visibility *string `json:"visibility,omitempty"`
	Settings *string `json:"settings,omitempty"`
	CreatorId *string `json:"creatorId,omitempty"`
	WorkspaceId *string `json:"workspaceId,omitempty"`
	CreatedAt *string `json:"createdAt,omitempty"`
	UpdatedAt *string `json:"updatedAt,omitempty"`
}
